test/json_texture_test.js

```diff
@@ -0,0 +1,28 @@
+var assert = require('assert');
+var configUtils = require('./utils/config');
+var testUtils = require('./utils/test');
+var fs = require('fs');
+var pkg = require('../package.json');
+
+describe('A hash map of texture info, similar to TexturePacker', function () {
```

Comment:
From the perspective of BDD, we are referring to the initial context which is information about multiple sprites. We don't start talking about the generated result until we reach "process by `spritesheet-templates`".